### ISSUE TITLE
plot individual components of holidays

### ISSUE THREAD

**Issue by wjhrdy** [facebookincubator/prophet#60]
Having holidays as an aggregate view in prophet_plot_components is concise but there should maybe be an option to view holiday components individually.

Here is some code that if put in the prophet_plot_components function would separate out the holiday components.

 ```
 df.s <- cbind(ds = df$ds, df[, holiday.comps])
 df.s_tidy <- tidyr::gather_(df.s,key_col = "holiday", value = "effect", gather_cols = holiday.comps)
 gg.holidays <- ggplot2::ggplot(df.s_tidy, ggplot2::aes(x = ds, 
                                                    y = effect)) + 
                           ggplot2::geom_line(color = forecast.color, na.rm = TRUE) + 
                           ggplot2::facet_wrap(~ holiday, ncol = 1)
```

**Comment by seanjtaylor** [comment]
This is a nice idea.  We'll consider adding it as an enhancement for the 0.2 release.

**Comment by bletham** [comment]
https://github.com/facebookincubator/prophet/commit/8f1607cd93d4641d690d95c5f4dd2737e8e7152f adds a method `plot_forecast_component` which can be used to plot any individual holiday by name. I'm inclined to leave it at this for plotting individual holidays, since including them on the components plot would make that plot very long for more than a couple holidays. This hasn't made it to the R yet but will in #101.

**Comment by bletham** [comment]
The `plot_forecast_component` is now available in v0.2 on CRAN and pypi. It's described in the documentation here:
https://facebookincubator.github.io/prophet/docs/seasonality_and_holiday_effects.html

or of course in the docstring for the function.
